### PR TITLE
replace "client" by "initiator" (fate#319293), bnc#920380, bnc#977173

### PR DESCRIPTION
**Pull request by gabi2**

### CODE REVIEW

**Comment by jsrain** [comment]
LGTM

File: src/modules/IscsiLioData.rb [hunk 1]
```diff
@@ -445,10 +445,14 @@ def SetNetworkPortal(tgt, tpg, np, new_port, add_all)
...
-        Builtins.foreach(m) do |tpg, dummy|
-          ret = Builtins.add(ret, [key, tpg])
-          Builtins.y2milestone("GetTargets key:%1 tpg:%2", key, tpg)
+      target_info = @data["tgt"] || {}
+
+      target_info.each do |target, data|
+        data.each do |tpg, tpg_info|
+          end_point = tpg_info["ep"] || { }
+          status = end_point["enabled"] || false
+          ret = ret << [target, tpg, status]
```
Comment (ancorgs) [inline src/modules/IscsiLioData.rb]: Surplus assigning, this should be only
```
ret << [target, tpg, status]
```

File: src/include/iscsi-lio-server/widgets.rb [hunk 2]
```diff
@@ -775,24 +775,26 @@ def initTable(key)
...
           )
         )
         Builtins.y2milestone(
-          "tgt:%1 tpg:%2 ctgt:%3 ctpg:%4",
-          Ops.get_string(l, 0, ""),
-          Ops.get_integer(l, 1, 0),
+          "tgt:%1 tpg:%2 enabled:%3 ctgt:%4 ctpg:%5",
+          target[0] || "",
+          target[1] || 0,
+          target[2] || false,
```
Comment (ancorgs) [inline src/include/iscsi-lio-server/widgets.rb]: NP: A little bit too much defensive, but better safe than sorry :-)

**Comment by ancorgs** [comment]
In general, LGTM, but it would be better to fix the surplus assignment (strictly speaking it works, but...)

**Pull request merged**